### PR TITLE
feat: :sparkles: Notifications

### LINKED ISSUE #32: Notification when files uploaded
Or a sound effect, but nowadays browsers make that a permission too

Stopgap for #7

### PR DESCRIPTION
**Pull request by Jack5079**
button isn't as polished as i'd like

Closes #32

### CODE REVIEW

File: src/svelte/elem/UploadWindow.svelte [hunk 1]
```diff
@@ -25,50 +25,86 @@
...
+                        }).addEventListener(
+                            "notificationclick",
+                            ({ action }) => {
+                                if (action === "open") {
+                                    open(
+                                        "/download/" +
+                                            uploads[x].uploadStatus.fileId
+                                    )
+                                } else if (action == "delete") {
+                                    alert("TODO")
```
Comment (Jack5079) [inline src/svelte/elem/UploadWindow.svelte:93]: yeah i need to do this

Comment (Jack5079) [reply src/svelte/elem/UploadWindow.svelte:93]: or you @nbitzz idk

Comment (Jack5079) [reply src/svelte/elem/UploadWindow.svelte:93]: or should this even be in a notif /shrug

Comment (nbitzz) [reply src/svelte/elem/UploadWindow.svelte:93]: probably not i don't know why you would want to delete a file immediately after upload

Comment (nbitzz) [inline src/svelte/elem/UploadWindow.svelte:93]:
```suggestion
```

**Review by nbitzz — CHANGES_REQUESTED** [review]

**Review by nbitzz — APPROVED** [review]

**Comment by Jack5079** [comment]
charlie is not a frontend person

**Comment by Jack5079** [comment]
😠 That is not a notif problem.

**Pull request merged**